projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -53,7 +56,15 @@ export default function ProductManagementInspectorControl( {
 		event.preventDefault();
 		setApiState( API_STATE_REQUESTING );
 		saveProduct(
-			{ title, currency, price, interval },
+			{
+				title,
+				currency,
+				price,
+				interval,
+				type: isMarkedAsDonation ? 'donation' : null,
+				buyer_can_change_amount: isCustomAmount,
```

Comment:
Upon successfully adding the plan, should these two form fields be reset to their default values? (see callback on line 72)

Reply:
https://github.com/Automattic/jetpack/issues/23575 is similar